src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTestTransformer.java

```diff
@@ -73,11 +76,7 @@
   private final PhpFeatureConfig featureConfig = new PhpFeatureConfig();
   private final MockServiceTransformer mockServiceTransformer = new MockServiceTransformer();
 
-  private final PackageMetadataConfig packageConfig;
-
-  public PhpGapicSurfaceTestTransformer(PackageMetadataConfig packageConfig) {
-    this.packageConfig = packageConfig;
-  }
+  public PhpGapicSurfaceTestTransformer() {}
```

Comment:
We could remove this constructor right? Or is it here for readability?

Reply:
There was no particular reason (using automatically generated one is equally fine, I believe). I can remove it.